pkg/actors/targets/workflow/activity/execute.go

```diff
@@ -112,7 +112,10 @@ func (a *activity) executeActivity(ctx context.Context, name string, taskEvent *
 	// send completed event to orchestrator wf actor
 	wfActorType := a.workflowActorType
 	if router := taskEvent.GetRouter(); router != nil {
-		wfActorType = a.actorTypeBuilder.Workflow(router.GetSourceAppID())
```

Comment:
It should _not_ be the case that `SourceAppID` is an empty string. Clearly there is something wrong inside of durabletask, which is where this fix should be. It cannot be the case that both `router` is not nil, and this value is empty.

Reply:
yes, I'm trying to see where the root cause is at, like was it changed in python durable task, or go durable task.  when testing from python directly against go durable task it works fine, but when testing against dapr it fails, and why I try to find issues in dapr instead of durable task.